### ISSUE TITLE
Shellcheck is fantastic and I want to buy you a beer

### ISSUE THREAD

**Issue by benliddicott** [koalaman/shellcheck#2086]
#### For new checks and feature suggestions
- [X] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/i/want/to/buy/you/a/beer
echo "what is your patreon/paypal/etc"

```

#### Here's what shellcheck currently says:
Nothing

#### Here's what I wanted or expected to see:
Some way to buy you a beer.

Shellcheck is fantastic and I want to buy you a beer.
If you have a patreon/paypal/whatever let me know and I will do so.

**Comment by benliddicott** [comment]
Seriously, everyone who writes shellscripts should use it, **especially** newbies, because it's basically a built-in tutor which will enable you to avoid many common errors.

Shellcheck finds bugs. Do you want bugs? No, you do not. So use it.

**Comment by koalaman** [comment]
I'm so happy you're finding the tool useful!

The community response and traction has been rewarding enough. I did enable the [github sponsorship option](https://github.com/sponsors/koalaman) at some point, but i don't know whether it allows once-off donations. If not, then just telling your newbie friends is plenty.

Thanks!

**Comment by benliddicott** [comment]
I was not aware there was such a thing as a github sponsorship option. Done.
